pytd/pandas_td/ipython.py

```diff
@@ -1,10 +1,10 @@
 """IPython Magics
 
-    IPython magics to access to Treasure Data. Load the magics first of all:
+IPython magics to access to Treasure Data. Load the magics first of all:
```

Comment:
Not sure why the linter decided to remove the tab here?

Reply:
I think the linter is correct though. Multiline comments usually align with the `"""`, not the text